From linux-wireless-owner@vger.kernel.org Thu Jan 26 18:23:33 2012
Return-Path: linville@tuxdriver.com
Received: from zmta02.collab.prod.int.phx2.redhat.com (LHLO
 zmta02.collab.prod.int.phx2.redhat.com) (10.5.5.32) by
 zmail15.collab.prod.int.phx2.redhat.com with LMTP; Thu, 26 Jan 2012
 18:23:33 -0500 (EST)
Received: from localhost (localhost.localdomain [127.0.0.1]) by
 zmta02.collab.prod.int.phx2.redhat.com (Postfix) with ESMTP id 3BF74128BE0
 for <linville@redhat.com>; Thu, 26 Jan 2012 18:23:33 -0500 (EST)
Received: from zmta02.collab.prod.int.phx2.redhat.com ([127.0.0.1]) by
 localhost (zmta02.collab.prod.int.phx2.redhat.com [127.0.0.1])
 (amavisd-new, port 10024) with ESMTP id 130PQ7pzqAEV for
 <linville@redhat.com>; Thu, 26 Jan 2012 18:23:33 -0500 (EST)
Received: from int-mx09.intmail.prod.int.phx2.redhat.com
 (int-mx09.intmail.prod.int.phx2.redhat.com [10.5.11.22]) by
 zmta02.collab.prod.int.phx2.redhat.com (Postfix) with ESMTP id 24E00128BDF
 for <linville@mail.corp.redhat.com>; Thu, 26 Jan 2012 18:23:33 -0500 (EST)
Received: from mx1.redhat.com (ext-mx11.extmail.prod.ext.phx2.redhat.com
 [10.5.110.16]) by int-mx09.intmail.prod.int.phx2.redhat.com (8.14.4/8.14.4)
 with ESMTP id q0QNNXuF014708 for <linville@redhat.com>; Thu, 26 Jan 2012
 18:23:33 -0500
Received: from smtp.tuxdriver.com (charlotte.tuxdriver.com [70.61.120.58])
 by mx1.redhat.com (8.14.4/8.14.4) with ESMTP id q0QNNWYF026587 for
 <linville@redhat.com>; Thu, 26 Jan 2012 18:23:32 -0500
Received: from sapphire.tuxdriver.com ([70.61.120.61] helo=localhost) by
 smtp.tuxdriver.com with esmtp (Exim 4.63) (envelope-from
 <linville@tuxdriver.com>) id 1RqYfD-0005zt-9K for linville@redhat.com; Thu,
 26 Jan 2012 18:23:31 -0500
Resent-From: "John W. Linville" <linville@tuxdriver.com>
Resent-Date: Thu, 26 Jan 2012 18:23:12 -0500
Resent-Message-ID: <20120126232312.GB2190@tuxdriver.com>
Resent-To: linville@redhat.com
Received: from vger.kernel.org ([209.132.180.67]) by smtp.tuxdriver.com
 with esmtp (Exim 4.63) (envelope-from
 <linux-wireless-owner@vger.kernel.org>) id 1RqYEt-0005tm-3A for
 linville@tuxdriver.com; Thu, 26 Jan 2012 17:56:21 -0500
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand id
 S1752042Ab2AZWzw (ORCPT <rfc822;linville@tuxdriver.com>); Thu, 26 Jan 2012
 17:55:52 -0500
Received: from server19320154104.serverpool.info ([193.201.54.104]:52032
 "EHLO hauke-m.de" rhost-flags-OK-OK-OK-OK) by vger.kernel.org with ESMTP id
 S1751796Ab2AZWzv (ORCPT <rfc822;linux-wireless@vger.kernel.org>); Thu, 26
 Jan 2012 17:55:51 -0500
Received: from localhost (localhost [127.0.0.1]) by hauke-m.de (Postfix)
 with ESMTP id E6CFE8F61; Thu, 26 Jan 2012 23:55:50 +0100 (CET)
X-Virus-Scanned: Debian amavisd-new at hauke-m.de 
Received: from hauke-m.de ([127.0.0.1]) by localhost (hauke-m.de
 [127.0.0.1]) (amavisd-new, port 10024) with ESMTP id HGH3hmHAsDa7; Thu, 26
 Jan 2012 23:55:47 +0100 (CET)
Received: from localhost.localdomain (unknown [134.102.132.222]) by
 hauke-m.de (Postfix) with ESMTPSA id F1F248F60; Thu, 26 Jan 2012 23:55:46
 +0100 (CET)
From: Hauke Mehrtens <hauke@hauke-m.de>
To: mcgrof@gmail.com, mcgrof@qca.qualcomm.com
Cc: linux-wireless@vger.kernel.org, Hauke Mehrtens <hauke@hauke-m.de>
Subject: [PATCH] compat-wireless: fix some config options
Date:	Thu, 26 Jan 2012 23:55:44 +0100
Message-Id: <1327618544-23163-1-git-send-email-hauke@hauke-m.de>
Sender: linux-wireless-owner@vger.kernel.org
Precedence: bulk
List-ID: <linux-wireless.vger.kernel.org>
X-Mailing-List:	linux-wireless@vger.kernel.org
X-Spam-Status: No
X-RedHat-Spam-Score: -0.01  (T_RP_MATCHES_RCVD)
X-Scanned-By: MIMEDefang 2.68 on 10.5.11.22
X-Scanned-By: MIMEDefang 2.68 on 10.5.110.16
X-Evolution-Source: imap://linville@mail.corp.redhat.com/
Content-Transfer-Encoding: 8bit
Mime-Version: 1.0

Adds some missing config options interceded in the last time and remove
some old now unused config options.

Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>
---
 config.mk |    8 +++++---
 1 files changed, 5 insertions(+), 3 deletions(-)

diff --git a/config.mk b/config.mk
index 4d2ad5c..9433ce9 100644
--- a/config.mk
+++ b/config.mk
@@ -281,6 +281,8 @@ CONFIG_ATH9K_COMMON=m
 # for long range considerations.
 CONFIG_ATH9K_RATE_CONTROL=y
 
+CONFIG_ATH9K_BTCOEX_SUPPORT=y
+
 # PCI Drivers
 ifdef CONFIG_PCI
 
@@ -288,10 +290,10 @@ CONFIG_ATH5K_PCI=y
 CONFIG_ATH9K_PCI=y
 
 CONFIG_IWLWIFI=m
+CONFIG_IWLWIFI_P2P=y
 # CONFIG_IWLWIFI_DEBUG=y
 # CONFIG_IWLWIFI_DEBUGFS=y
 # CONFIG_IWLWIFI_DEVICE_TRACING=y
-# CONFIG_IWLWIFI_DEVICE_SVTOOL=y
 # CONFIG_IWLWIFI_DEBUG_EXPERIMENTAL_UCODE=y
 
 CONFIG_IWLEGACY=m
@@ -314,7 +316,6 @@ CONFIG_B43_PHY_LP=y
 CONFIG_B43_PHY_N=y
 CONFIG_B43_PHY_HT=y
 # CONFIG_B43_PHY_LCN=y
-# CONFIG_B43_FORCE_PIO=y
 # CONFIG_B43_DEBUG=y
 
 CONFIG_B43LEGACY=m
@@ -375,7 +376,7 @@ CONFIG_BCMA_BLOCKIO=y
 CONFIG_BCMA_HOST_PCI=y
 # CONFIG_BCMA_DEBUG=y
 CONFIG_B43_BCMA=y
-CONFIG_B43_BCMA_PIO=y
+CONFIG_B43_BCMA_EXTRA=y
 
 CONFIG_P54_PCI=m
 
@@ -614,6 +615,7 @@ endif #CONFIG_COMPAT_KERNEL_2_6_27
 
 ifndef CONFIG_COMPAT_KERNEL_2_6_29
 CONFIG_BRCMFMAC=m
+CONFIG_BRCMFMAC_SDIO=y
 endif #CONFIG_COMPAT_KERNEL_2_6_29
 
 endif #CONFIG_MMC
-- 
1.7.5.4

--
To unsubscribe from this list: send the line "unsubscribe linux-wireless" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html


